### ISSUE TITLE
Bug report - [Illegal state, command line failed with 127]

### ISSUE THREAD

**Issue by NPastorale** [CypherpunkArmory/UserLAnd#945]
# Describe the bug

Whenever trying to create a Session, it fails while extracting because of insufficient permissions with error 127
# Steps to reproduce the behavior:
1. Tap on any Distribution
2. Enter the required information
3. After the file downloads finish and the app starts to extract them, it failes due to permissions

# Screenshots

![Screenshot 2019-07-15 at 22 30 55](https://user-images.githubusercontent.com/7849710/61247355-2e26e000-a751-11e9-8ab9-973ed0a6c2d3.png)

![Screenshot 2019-07-15 at 22 38 46](https://user-images.githubusercontent.com/7849710/61247431-54e51680-a751-11e9-80e0-fb3a3b7271e2.png)

# Device Information

Device: Asus Chromebook Flip C302 (Cave)
Android Version: 7.1.1
UserLAnd Version: 2.5.15
Running ChromeOS in Developer Mode

--- Final thoughts ---
I don't actually know if ChromeOS is supported, I came across UserLAnd just yesterday while looking for options to run VSCode and PyCharm cleaner than with Crouton.
Let me know if I can be of any more help.
Thanks!!

**Comment by MatthewTighe** [comment]
Does the progress dialog spin with that message for a while before the illegal state message appears? I wouldn't expect you to be able to get that second screenshot

**Comment by NPastorale** [comment]
Yes, first it goes trough "Downloading required assets...", then it shows the pop up message about the illegal state and after that it keeps spinning with the "Permission denied" until I close the app

**Comment by MatthewTighe** [comment]
Ah, that makes sense. I believe I might have access to a Chromebook, so if I can get a hold of it I should be able to debug this issue.

**Comment by NPastorale** [comment]
Great! Let me know if I can be of help. You may remote into my chromebook if you think that would be useful

**Comment by MetaNite64** [comment]
Can confirm that this is still an issue on Chrome OS 78 (Android 7.1.1, latest UserLAnd as of Nov 26)

**Comment by corbinlc** [comment]
Closing as part of a repository cleanup. If this issue is still relevant, feel free to reopen it or file a fresh one.
